doc/rtd/topics/network-config-format-v1.rst

```diff
@@ -124,6 +129,11 @@ bond interfaces.  Specifying a MAC Address is optional.  If ``mac_address`` is
 not present, then the bond will use one of the MAC Address values from one of
 the bond interfaces.
 
+.. note:
```

Comment:
What do you think about using [`.. include:`](https://docutils.sourceforge.io/docs/ref/rst/directives.html#include) to avoid maintaining this wording in these various places?

Reply:
As mentioned on IRC it seems that RST include can only include separate files - not sure if its worth creating a separate file just for the 3 places that have this note.